### PR TITLE
added target reference, doc fix and tests for vni

### PR DESCRIPTION
**Pull request by ujjwal-ibm**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
=== RUN   TestAccIBMISSubnetReservedIP_targetVni
--- PASS: TestAccIBMISSubnetReservedIP_targetVni (120.68s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     122.845s
```
```
=== RUN   TestAccIBMISSecurityGroupTargetDataSource_vni
--- PASS: TestAccIBMISSecurityGroupTargetDataSource_vni (113.65s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     115.783s
```
```
=== RUN   TestAccIBMISSecurityGroupTargets_vni
--- PASS: TestAccIBMISSecurityGroupTargets_vni (114.77s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     117.489s
```

### CODE REVIEW

File: website/docs/d/is_subnet_reserved_ips.html.markdown [hunk 1]
```diff
@@ -50,5 +50,15 @@ In addition to the argument reference list, you can access the following attribu
   - `resource_type` -  (String) The resource type.
   - `target` - (String) The ID of the target for the reserved IP.
   - `target_crn` - (String) The crn of the target for the reserved IP.
+	- `target_reference` - (List) The target this reserved IP is bound to. If absent, this reserved IP is provider-owned or unbound.
```
Comment (sumitkumartiwari) [inline website/docs/d/is_subnet_reserved_ips.html.markdown:53]: Update the formatting for `target_reference`

**Review by deepaksibm — APPROVED** [review]

File: website/docs/d/is_subnet_reserved_ips.html.markdown (hunk 1 above)
Comment (ujjwal-ibm) [reply website/docs/d/is_subnet_reserved_ips.html.markdown:53]: resolved

Comment (sumitkumartiwari) [reply website/docs/d/is_subnet_reserved_ips.html.markdown:53]: Is this `target_reference` is under `target_crn` ? 

Check this and let me know ?

Comment (ujjwal-ibm) [reply website/docs/d/is_subnet_reserved_ips.html.markdown:53]: no

Comment (ujjwal-ibm) [reply website/docs/d/is_subnet_reserved_ips.html.markdown]: updated the file

**Review by sumitkumartiwari — APPROVED** [review]

**Pull request merged**